autokeras/hypermodel/preprocessor.py

```diff
@@ -253,5 +244,23 @@ def transform(self, x):
     def output_types(self):
         return tf.float64,
 
+    @property
     def output_shape(self):
         return self._shape
+
+    def get_weights(self):
```

Comment:
These are not "weights" (numerical arrays), so prefer using a different name, e.g. save/load or get_state / set_state